### DISCUSSION TITLE
Follows from in Jaeger

### DISCUSSION THREAD

**Discussion by mTsBucy1** [tokio-rs/tracing-opentelemetry#181] [Q&A]
Hi, I have a function like this in my code:
```rust
#[tracing::instrument(level = "TRACE", follows_from = [cause], skip(self))]
pub async fn handle_task(&self, task: Task, cause: Option<tracing::span::Id>) { .. }
```

However, I can't seem to see this cause nicely in Jaeger. According to the discussion at https://github.com/jaegertracing/jaeger-ui/issues/662#issuecomment-732359567, I believe there should be an indicator, but I am not sure.
I do see the tag `cause = Some(Id(6755399441055774))` along with the other fields.

So my question is, whether this is a bug, that the information is not passed properly via OTLP to the backend, or whether this is a problem of jaeger, or maybe if this is intentional, or something else entirely. Thanks :)

**Comment by djc** [comment]
Sorry, I won't have time to look into this (on a volunteer basis). If you'd like to discuss commercial support, feel free to send me an email.

**Reply by mTsBucy1** [reply]
fair
